websites/H/HentaiVN/presence.ts

```diff
@@ -0,0 +1,570 @@
+const presence = new Presence({
+		clientId: "942782020055089192"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let page = "1",
```

Comment:
Variable is never redefined and is always the same value?

Reply:
predefined variable if there is no query "page" in query